### PR TITLE
Additional cleanup in apply_role

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by nicolasbock** [comment]
+1

**Comment by toabctl** [comment]
+1

File: crowbar_framework/app/models/service_object.rb [hunk 1]
```diff
@@ -1016,11 +1016,7 @@ def apply_role(role, inst, in_queue)
 
     # We'll build an Array where each item represents a batch of work,
     # and the batches must be performed sequentially in this order.
-    # This will mirror the ordering specified by element_order below,
```
Comment (aspiers) [inline crowbar_framework/app/models/service_object.rb]: Why did you remove this comment?
